sqlx-cli/src/opt.rs

```diff
@@ -7,6 +7,10 @@ use clap_complete::Shell;
 #[derive(Parser, Debug)]
 #[clap(version, about, author)]
 pub struct Opt {
+    /// Do not automatically load `.env` files.
+    #[clap(long, default_value = "false")]
```

Comment:
```suggestion
    #[clap(long)]
```

`default_value = "false"` is redundant for a boolean flag.